cmd/modelcmd/clientstore.go

```diff
@@ -43,20 +43,20 @@ func (s QualifyingClientStore) QualifiedModelName(controllerName, modelName stri
 
 // Implements jujuclient.ModelGetter.
 func (s QualifyingClientStore) ModelByName(controllerName, modelName string) (*jujuclient.ModelDetails, error) {
-	modelName, err := s.QualifiedModelName(controllerName, modelName)
+	qualifiedModelName, err := s.QualifiedModelName(controllerName, modelName)
```

Comment:
These seem needlessly wordy - it's clear that it's the qualified name by the method being called, and this method's only a few lines long anyway (so if the reader wasn't sure whether the name was qualified they could just look up). (Sorry, bike-sheddy, feel free to ignore!)

Reply:
I needed a name != "modelName" as that was what the bug was - the local var was masking the function param with the same name